### DISCUSSION TITLE
select tools on charts

### DISCUSSION THREAD

**Discussion by snarum** [datalust/seq-tickets#1632] [Help]
Just out of curiosity: What is the purpose of the select tools (Rectangle and Lasso select) on the charts?

**Comment by nblumhardt** [comment]
Thanks for dropping us a line, @snarum; you've spotted a bug, those two tools should be hidden, Seq doesn't currently provide any additional functionality through these (I suspect a CSS or configuration regression has caused these to show up).

I'll raise an issue tracker ticket for this.

Thanks again!
